lib/utils.js

```diff
@@ -26,3 +26,13 @@ export const parents = (test, count = 0) => {
   }
   return parents(test.parent, ++count);
 };
+
+export const UTCDateTime = new Intl.DateTimeFormat('en-AU', {
```

Comment:
Why are we using `en-AU` instead of `en-US`?

Reply:
This is intentional, this formatter comes from Respec originally: https://github.com/w3c/respec/blob/471d6f5b20aab17e0b9db969418ef53f4b568d4c/src/core/utils.js#L33-L42

I couldn't find a spec for w3c spec that is being referred to here, as there a lot of w3c spec related to datetime.